drivers/SmartThings/zigbee-switch/src/configurations.lua

```diff
@@ -60,8 +63,29 @@ local devices = {
   }
 }
 
+
 local configurations = {}
 
+configurations.power_reconfig_wrapper = function(orig_function)
```

Comment:
Is orig_function supposed to be called from this function?

Reply:
lol, yes